pyproject.toml

```diff
@@ -1,8 +1,16 @@
 [tool.poetry]
-name = "sed"
-version = "0.1.0"
+name = "sed-processor"
+packages = [
+    {include = "sed"}
+]
+version = "0.1.5a2"
 description = "Single Event Data Frame Processor: Backend to handle photoelectron resolved datastreams"
-authors = ["Muhammad Zain Sohail <zainsohail04@gmail.com>"]
+authors = ["OpenCOMPES team <zainsohail04@gmail.com>"]
```

Comment:
is it possible to add the main authors explicitly, @zain-sohail @rettigl and @steinnymir ? It would be nice if we all appear in the pypi page. I'm not sure if this is happening here though.

Reply:
Of course it is. But I think we decided to use this at some point. Can also just put the names
I can easily add everyone to the maintainers in PyPi. I just need your PyPI account email addresses.

Reply:
invited

Reply:
accepted. thanks!